stable/cluster-autoscaler/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+description: Scales worker nodes within autoscaling groups.
+icon: https://github.com/kubernetes/kubernetes/blob/master/logo/logo.png
+name: cluster-autoscaler
+version: 7.1.0
+appVersion: 1.17.1
+home: https://github.com/kubernetes/autoscaler
+sources:
+- https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler
+- https://github.com/spotinst/kubernetes-autoscaler/tree/master/cluster-autoscaler
+maintainers:
```

Comment:
Are these upstream maintainers?

If so, this is a `stable/` chart so we need to add our own maintainers so as to not confuse our end-users.

Reply:
Yes, these are upstream maintainers @shaneutt . Should I replace completely maintainers section or just extend it with our names?

Reply:
@shaneutt we are gonna apply changes to this vanilla chart, so I'd say we could add them as part of a different PR, imho.

Reply:
> Yes, these are upstream maintainers @shaneutt . Should I replace completely maintainers section or just extend it with our names?

For now, replace it with our names. We should not claim those people are responsible for our fork.